consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/IndexController.java

```diff
@@ -30,12 +32,16 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.Optional;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
 /** An index controller class to balance the performance degradation of frequent disk I/O. */
 @ThreadSafe
 public class IndexController {
 
+  public static final String separator = "-";
```

Comment:
uppercase ？

Reply:
I just follow the `File.separator` style. Maybe this is trivial?
<img width="841" alt="image" src="https://user-images.githubusercontent.com/32640567/207559815-db1038ea-aea0-49f7-a614-50c354836704.png">